modules/ROOT/pages/domain-based-application-cookbook.adoc

```diff
@@ -0,0 +1,114 @@
+= Test Domain-Based Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MUnit 2.2 allows you to write MUnit test suites for your Mule applications that share their resources with a Mule domain without the need for any extra configurations.
```

Comment:
Maybe we should change this to 2.x instead of 2.2, since the feature is present in any version of MUnit 2, and otherwise we would need to change this each time a new version comes out.